Utils/Dataflow/069_upload2es/load_data.sh

```diff
@@ -57,23 +99,34 @@ load_files () {
   do
     ${cmd}${INPUTFILE} || exit 3
   done
+  echo -ne "$EOProcess"
 }
 
 load_stream () {
   log "Switched to the stream mode."
-  while read -r -d "$DELIMITER" line; do
+  while read -r -d $(echo -ne "$EOBatch") line; do
     n=`ps axf | grep '[c]url' | grep "$HOST:$PORT" | wc -l`
     while [ $n -gt $CURL_N_MAX ]; do
       sleep $SLEEP
       n=`ps axf | grep '[c]url' | grep "$HOST:$PORT" | wc -l`
     done
     echo "$line" | ${cmd}- &
-    echo -n "$EOProcess"
+    echo -ne "$EOProcess"
   done
 }
 
 if [ -z "$1" ] ; then
+  if [ -z "$EOP" ] ; then
+    EOProcess="\0"
+  else
+    EOProcess="$EOP"
```

Comment:
I\`m afraid that here will also appear that issue with empty string passed as EOP (`-E ''`).